scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
```

Comment:
When using something yes, usability is more important. However in code the readability outweighs the need for it to be 3 chars. If you were the only person reading this code It would be fine. But you aren't, others need to be able to look at it and quickly get a glance at what it does. I'm not asking you to turn it into a massive thing. But 3 char is not enough for someone not familiar with this piece of code to understand what it does at a glance. What you name it doesn't matter to me as long as I can understand what it does by name, and as long as its relevent and follows the rules me and you have discussed in the past (i.e not a joke)